daprdocs/content/en/reference/components-reference/supported-secret-stores/hashicorp-vault.md

```diff
@@ -64,7 +64,7 @@ The above example uses secrets as plain strings. It is recommended to use a loca
 | vaultKVPrefix | N | The prefix in vault. Defaults to `"dapr"` | `"dapr"`, `"myprefix"` |
 | vaultKVUsePrefix | N | If false, vaultKVPrefix is forced to be empty. If the value is not given or set to true, vaultKVPrefix is used when accessing the vault. Setting it to false is needed to be able to use the BulkGetSecret method of the store.  | `"true"`, `"false"` |
 | enginePath | N | The [engine](https://www.vaultproject.io/api-docs/secret/kv/kv-v2) path in vault. Defaults to `"secret"` | `"kv"`, `"any"` |
-| vaultValueType | N | Vault value type. `map` means to parse the value into `map[string]string`, `text` means to use the value as a string. Defaults to `"map"` | `"map"`, `"text"` |
+| vaultValueType | N | Vault value type. `map` means to parse the value into `map[string]string`, `text` means to use the value as a string. The latter will make Vault behave as a secret store with name/value semantics. Defaults to `"map"` | `"map"`, `"text"` |
```

Comment:
```suggestion
| vaultValueType | N | Vault value type. `map` means to parse the value into `map[string]string`, `text` means to use the value as a string. 'map' sets the `multipleKeyValuesPerSecret` behavior. `text' makes Vault behave as a secret store with name/value semantics.  Defaults to `"map"` | `"map"`, `"text"` |
```

Reply:
One thing: Up until this change, we never referred to the behavior by the feature flag name (`multipleKeyValuesPerSecret`). The current docs always use the full _"multiple key-value per secret behaviour"_.